files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -681,3 +681,23 @@
   when:
     - variant == "internal-buildserver"
     - not ansible_is_chroot
+
+# CIS: Set default umask (DLPX-86533)
+# We need to set default umask as 022 in the /etc/profile and /etc/bash.bashrc files,
+# so that the same can be applied for all the users on the engine.
+- blockinfile:
+    path: /etc/profile.d/set-umask-for-all-users.sh
+    create: yes
+    block: |
+      #!/bin/sh
+      umask 022
+
+- file:
+    path: /etc/profile.d/set-umask-for-all-users.sh
```

Comment:
This really should be delivered as a stand-alone file rather than constructed in an ansible task (e.g. `files/common/etc/profile.d/set-umask-for-all-users.sh`). The reason is primarily that once this logic is there, if we ever need to change the contents of the file, the ansible logic becomes more complex to handle upgrade cases. If you have to modify an existing file, then doing that in ansible seems like the logical choice, but in this case, this file doesn't exist yet...

Reply:
Sure, I have moved it to a separate file at `files/common/etc/profile.d/set-umask-for-all-users.sh`